benchmarks/relay_constellation/verifier/models.py

```
@@ -0,0 +1,225 @@
+    max_eccentricity: float | None
+    min_inclination_deg: float | None
+    max_inclination_deg: float | None
+    max_isl_range_m: float
+    max_links_per_satellite: int
+    max_links_per_endpoint: int
+    max_ground_range_m: float | None
+
+    @property
+    def total_samples(self) -> int:
+        horizon_seconds = (self.horizon_end - self.horizon_start).total_seconds()
+        return int(round(horizon_seconds / self.routing_step_s))
```

Ruff (0.15.9)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)